<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>&amp;Rules</string>
    </property>
    <addaction name="actionDelete_selected"/>
    <addaction name="separator"/>
    <addaction name="actionShow_active_only"/>
    <addaction name="actionShow_all"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Iptable</string>
    </property>
    <addaction name="actionAdd_New_Rules"/>
    <addaction name="actionShow_added_Iptable_rules"/>
   </widget>
   <widget class="QMenu" name="menuCustom">
    <property name="title">
     <string>custom</string>
    </property>
    <addaction name="actionSSH_filter_builder"/>
    <addaction name="actionFTP_filter_builder"/>
    <addaction name="actionDNS_filter_builder"/>
    <addaction name="actionWEB_filter_builder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRules"/>
   <addaction name="menuAdd"/>
   <addaction name="menuCustom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="text">
    <string>&amp;Delete selected</string>
   </property>
  </action>
  <action name="actionShow_active_only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Show active only</string>
   </property>
  </action>
  <action name="actionShow_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show &amp;all</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save all to Rulesfile</string>
   </property>
  </action>
  <action name="actionPreferences_2">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionProcess_ID">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Process ID</string>
   </property>
  </action>
  <action name="actionPermission">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;ermission</string>
   </property>
  </action>
  <action name="actionFull_path">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Full path</string>
   </property>
  </action>
  <action name="actionIN_allowed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IN &amp;allowed</string>
   </property>
  </action>
  <action name="actionIN_denied">
   <property name="text">
    <string>IN &amp;denied</string>
   </property>
  </action>
  <action name="actionOUT_allowed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;OUT allowed</string>
   </property>
  </action>
  <action name="actionOUT_denied">
   <property name="text">
    <string>O&amp;UT denied</string>
   </property>
  </action>
  <action name="actionAdd_New_Rules">
   <property name="text">
    <string>Add Iptable Rule</string>
   </property>
  </action>
  <action name="actionShow_added_Iptable_rules">
   <property name="text">
    <string>Show added Iptable rules</string>
   </property>
  </action>
  <action name="actionSSH_filter_builder">
   <property name="text">
    <string>SSH filter builder</string>
   </property>
  </action>
  <action name="actionFTP_filter_builder">
   <property name="text">
    <string>FTP filter builder</string>
   </property>
  </action>
  <action name="actionDNS_filter_builder">
   <property name="text">
    <string>DNS filter builder</string>
   </property>
  </action>
  <action name="actionWEB_filter_builder">
   <property name="text">
    <string>WEB filter builder</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionProcess_ID</sender>
   <signal>changed()</signal>
   <receiver>actionProcess_ID</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
